api/HTMLInputElement.json

```diff
@@ -3005,7 +3005,10 @@
             "chrome": {
               "version_added": "7"
             },
-            "chrome_android": "mirror",
+            "chrome_android": {
+              "version_added": "18",
+              "notes": "While this property is recognized, the File API does not actually allow directory selection. See <a href='https://crbug.com/40248532'>bug 40248532</a>."
```

Comment:
There's an argument for also setting this to "partial": true. Thoughts?

Reply:
I think we really should add `"version_added": false,` with a note `"The property is recognized, but has no effect."` (what does it mean for the property being recognized), like here:

https://github.com/mdn/browser-compat-data/blob/28f6e138a3ebfd3c130b686ee8732cb0fe54528c/api/Notification.json#L652-L656

Reply:
https://codepen.io/danieljacobs/pen/bGXEJzd is what I meant.

Reply:
I'm worried if I change it to false again it will be changed back by an update using the https://mdn-bcd-collector.gooborg.com/tests/api/HTMLInputElement/webkitdirectory test by @queengooborg

Reply:
@queengooborg Is there a way to disable this test? I didn't find anything about it in the collector docs.

Reply:
I have done this. Hopefully it isn't reverted automatically.

Reply:
I just added the `needs collector update` label, so that this pops up.